lib/fragmentSender.js

```diff
@@ -57,36 +57,29 @@ module.exports = function(chat, fragments, payload, initialMessage = '', initial
             uri: urls.push(payload.push),
             json: true,
         }).then((push) => {
-            const nextReportIndex = push.reports.findIndex((r) => r.id === payload.report) + 1;
-            if (nextReportIndex > 0 && nextReportIndex < push.reports.length) {
-                quickReplies.push(
-                    quickReply(
-                        'Nächstes Thema',
-                        {
-                            action: 'report_start',
-                            push: payload.push,
-                            report: push.reports[nextReportIndex].id,
-                            type: payload.type,
-                        })
-                );
-            } else {
-                quickReplies.push(
-                    quickReply(
-                        'Reicht jetzt',
-                        {
-                            action: 'push_outro',
-                            push: payload.push,
-                        })
-                );
-            }
+            quickReplies.push(
+                quickReply(
+                    'Zur Übersicht',
+                    {
+                        action: 'current_news',
+                        intro: false,
+                    })
+            );
+            quickReplies.push(
+                quickReply(
+                    'Reicht jetzt',
+                    {
+                        action: 'push_outro',
+                        push: payload.push,
+                    })
+            );
 
             sendFragment(chat, messages, quickReplies);
 
             promMetrics.activity_push.inc({
                 timing: push.timing,
                 pushId: push.id,
                 reportId: payload.report,
-                reportIndex: nextReportIndex - 1,
```

Comment:
Please do not remove this metric label! This is needed and there will be a way to find out which report index we have sent